build.gradle

```diff
@@ -3,13 +3,14 @@ import org.jetbrains.kotlin.gradle.dsl.JvmTarget
 
 plugins {
     id "java"
-    id "org.jetbrains.kotlin.jvm" version "1.9.22"
+    id "org.jetbrains.kotlin.jvm" version "2.0.0"
     id "edu.wpi.first.GradleRIO" version "2024.3.2"
     id "com.peterabeles.gversion" version "1.10"
+    id 'com.google.devtools.ksp' version '2.0.0-1.0.21'
     id "idea"
 }
 
-def javaVersion = JavaVersion.VERSION_17
+def javaVersion = JavaVersion.VERSION_20
```

Comment:
Won't 22 work?

Reply:
Discuss with @Emma03L

Reply:
I don't really see a reason to use anything above 17

Reply:
I'd just use the newest version available and downgrade if it will be needed in the future.